dev/app.js

```diff
@@ -517,6 +534,16 @@ renderSiteApp(
                             pathname: "/panning/line/simple",
                             content: renderLineWithPanning,
                             title: "Simple"
+                        },
+                        {
+                            pathname: "/panning/line/y2-axis",
+                            content: renderLineY2AxisWithPanning,
+                            title: "Y2 Axis"
+                        },
+                        {
+                            pathname: "/panning/line/dynamic-data",
+                            content: renderLinePanningWithDynamicData,
+                            title: "Dynamic Data"
```

Comment:
Looks like you added dynamic data tab in dev site. can we have images regarding this just to see how it looks like?

Reply:
![Screenshot 2020-06-03 at 1 01 31 AM](https://user-images.githubusercontent.com/29273628/83561835-25402800-a536-11ea-8425-2984a9f8d9ea.png)

Reply:
I mean how graph looks like with dynamic data. if it is same as other graphs then that is fine i guess.

Reply:
The graph might not look same with dynamic data, as with this the graph recalculates the y-axis ranges. And if they are out of the current ranges, then it will update the y-axis to accommodate the ticks for the new data.

But this can be controlled by the user by setting the property `allowCallibration` as `false`.

More info on this property is available [here](https://github.com/cerner/carbon-graphs/blob/master/docs/core/Graph.md#optional)